src/compiler/types.ts

```diff
@@ -4899,6 +4902,9 @@ namespace ts {
         // Prefer aliases which are not directly visible
         UseAliasDefinedOutsideCurrentScope = 0x00000008,
 
+        // { [E.A]: 1 }
+        AllowComputedPropertyEnums = 0x00000009,

```

Comment:
I don't think this is right; these flags are powers of two (but not written as shifts for some reason; oversight?), so I don't think 9 is a good value. If the new one is the combination of other flags, then I would write it in terms of them.

Reply:
Okay. 
![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/509265/186214462-f363180f-7dc9-4a8c-a6ad-75ecfcfa9be6.gif)

Reply:
Not your oversight, ours... I truly don't know why this isn't a shift; I asked myself that same question the last time I read this code 😅